decidim-api/app/controllers/decidim/api/geocoder_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Api
+    class GeocoderController < Api::ApplicationController
+      def search
+        @search_string = params[:term]
```

Comment:
I think that we should look here for the configuration of Geocoder on Decidim.
Search for "Decidim.geocoder.present?" or "Decidim.geocoder.blank?" like we do on https://github.com/decidim/decidim/blob/4f6155fa785c50ffa54683556177b250c1b59f56/decidim-core/app/helpers/decidim/map_helper.rb#L27